terraform/gcp/db.tf

```diff
@@ -0,0 +1,36 @@
+# Create database
+resource "google_sql_database_instance" "d2p" {
+  name = "d2p"
+  database_version = "POSTGRES_15"
```

Comment:
I think we can go ahead and bump to 16. 
 https://cloud.google.com/blog/products/databases/postgresql-16-now-available-in-cloud-sql

Reply:
good catch!